packages/multichain/src/index.ts

```diff
@@ -1,9 +1,52 @@
-/**
- * Example function that returns a greeting for the given name.
- *
- * @param name - The name to greet.
- * @returns The greeting.
- */
-export default function greeter(name: string): string {
-  return `Hello, ${name}!`;
-}
+export {
+  getEthAccounts,
+  setEthAccounts,
+} from './adapters/caip-permission-adapter-eth-accounts';
+export {
+  getPermittedEthChainIds,
+  addPermittedEthChainId,
+  setPermittedEthChainIds,
+} from './adapters/caip-permission-adapter-permittedChains';
+
+export type { Caip25Authorization } from './scope/authorization';
+export { validateAndNormalizeScopes } from './scope/authorization';
+export {
+  KnownWalletRpcMethods,
+  KnownRpcMethods,
+  KnownWalletNamespaceRpcMethods,
+  KnownNotifications,
+  KnownWalletScopeString,
+} from './scope/constants';
+export type {
+  ExternalScopeString,
+  ExternalScopeObject,
+  ExternalScopesObject,
+  InternalScopeString,
+  InternalScopeObject,
+  InternalScopesObject,
+  ScopedProperties,
+  NonWalletKnownCaipNamespace,
+} from './scope/types';
+export { parseScopeString } from './scope/types';
+export {
+  isSupportedScopeString,
+  isSupportedAccount,
+  isSupportedMethod,
+  isSupportedNotification,
+} from './scope/supported';
```

Comment:
Nit: Maybe we could avoid exporting these? They aren't used in either draft extension PR it seems

```suggestion
```